### ISSUE TITLE
Group auto bid turns itself off!

### ISSUE THREAD

**Issue by stuboy333** [sweitzel/Biet-O-Matic-BE#27]
Firefox, windows x64
I just successfully bid on an item and was waiting for the next item which wasn't long after. 
I noticed with 15 or so seconds to go that "group auto bid" was red! 
I had not touched anything since the previous successful auction!
I re enabled it but it still did not bid, what is the minimum time it needs to re enable?
I think this has happened before as well.

It would be good to minimize the number of things you have to check in order to get BE to bid:- 
My idea would be simply putting a figure in the "Max Bid" box should automatically make it bid regardless of anything else on the page. 
I hope this is a good idea?

Hope your able to find the problem
Regards
Stuboy

**Comment by sweitzel** [comment]
Can you please provide some more information?

- Do you use multiple groups?
- Have you set the group where you observed the issue to "☝️ I want one" or " 🖐️ I want all"?

If you reenable group-autobid, it becomes active immediately.
To minimize things, you can currently only disable the groups completely. Then one auto-bid feature less.
Implementing a single autobid per item without a global autobid is a bit more complex to implement. Also it would need to be optional, as there are conflicting use cases (not everybody has the same requirements).

Thanks,
Sebastian

**Comment by stuboy333** [comment]
Hi Seb
No I never use groups, always want one.

here's the log for the item that it didnt bid for if it helps:-

Feb 20, 2021, 3:40:47 PM | Items | Updated | Price: 22 -> 46; Number of Bids: 13 -> 16; Status:  -> Bidding has ended on this item.
-- | -- | -- | --
Feb 20, 2021, 3:40:47 PM | Bidding | Status | Auction  not successful. Your bid was not received or accepted by eBay (e.g.  when another bidder placed a higher bid before yours).
Feb 20, 2021, 3:35:24 PM | Bidding | Info | Auto-bid is inactive for group Watch List
Feb 20, 2021, 3:25:47 PM | Items | Updated | Price: 19 -> 22; Number of Bids: 11 -> 13; Minimum Bid: 20 -> 23
Feb 20, 2021, 1:15:43 PM | Items | Updated | Picture:  https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l500.jpg ->  https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l400.jpg
Feb 20, 2021, 11:35:43 AM | Items | Updated | Item Maximum Bid: 47.33 -> 57.33
Feb 20, 2021, 9:15:44 AM | Items | Updated | Shipping  Cost: GBP 5.00 (approx. US $6.97) -> GBP 5.00 (approx. US $7.01);  Picture: https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l400.jpg  -> https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l500.jpg
Feb 19, 2021, 9:43:44 PM | Items | Updated | Price: 14.5 -> 19; Number of Bids: 9 -> 11; Minimum Bid: 15 -> 20
Feb 19, 2021, 6:46:12 PM | Items | Updated | Item Maximum Bid: 42.33 -> 47.33
Feb 19, 2021, 6:43:44 PM | Items | Updated | Price:  2.2 -> 14.5; Number of Bids: 3 -> 9; Minimum Bid: 2.4 -> 15;  Picture: https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l500.jpg  -> https://i.ebayimg.com/images/g/5QUAAOSw9PNgKpP0/s-l400.jpg
Feb 19, 2021, 4:43:45 PM | Items | Updated | Price: 1.2 -> 2.2; Number of Bids: 2 -> 3; Minimum Bid: 1.4 -> 2.4
Feb 19, 2021, 9:43:43 AM | Items | Updated | Shipping Cost: GBP 5.00 (approx. US $6.93) -> GBP 5.00 (approx. US $6.97)
Feb 18, 2021, 4:41:10 PM | Items | Updated | Shipping Cost: GBP 5.00 (approx. US $6.96) -> GBP 5.00 (approx. US $6.93)
Feb 16, 2021, 5:26:21 PM | Items | Updated | Item Auto-Bid: false -> true
Feb 16, 2021, 5:26:20 PM | Items | Updated | Item Maximum Bid: 42.33

I bid £57 so it should have bid

Am i able to disable groups? U advised me on version o.53 i think? that i had to enable groups because there was a bug if they were disabled!
Regards
Stu

**Comment by stuboy333** [comment]
OK I think i know why this is happening.
It is taking "watch list" as a group! Which it shouldn't as its the entire list!

Just confirmed the above is correct?
1) Could you confirm the bug is fixed so we can disable groups in the config again.
2) Make an exception for "watch group" else it will only ever bid on 1 item unless all times are in different groups
Regards

**Comment by sweitzel** [comment]
Lets focus on the main topic here, you are saying that the group auto bid turns itself off after an auction succeeds - which prevents further bidding for other items in the same group.
However with "☝️ I want one" active for a group, this is the expected behavior. It does not matter if you imported the items from the ebay watchlist or not, for BE its just a group like all other groups. 

Can you please explain why you expect that the Watch list items  group is different? Where does the confusion come from? What can I do to improve it?

**Comment by stuboy333** [comment]
ok main topic:-
I dont select any group, all items are under heading "watch list".
BE is treating "watchlist" as a group, because watch list is filled in when you import the items, its a default list title. 
All my items are in that group so it only bids on the first item in the list, if i win it switches off.
I think this is probably how you have programmed it, however maybe "Watch list" should be a special case that does not trigger the buy one to switch off?
Or is the solution to activate "buy many" to get the desired functionality? Even though many are all different items and in reality would each be in a separate group.
I hope that makes sense. If its the way you programmed it and how you want it to run then its not a fault it could just be confusing for the thick user like myself ;-)
Cheers

**Comment by sweitzel** [comment]
The feature to import items from watch list was implemented due to a user request. With it, you have an easy/efficient way to select potential interesting items in ebay which can later processed by BE. I agree that in a watchlist there are typically more than one group of items to be auctioned.
However the information about these groups inside the watchlist is not there.
You have to manually assign the items from the watchlist a different, more explicit group. Or alternatively select the "🖐️ I want all" option on the Watch list items groups.

So I do not see any issue here (definetly not a bug). The only possible improvement could be to set the default value of groups to "I want all", because that probably makes more sense. However then I think all groups should have this as default for consistency.

**Comment by stuboy333** [comment]
Yes I have had a think about it and i agree with you, its not a bug :-) 

Possibly changing the name of "i want all" to "bid on everything" and "i want 1" to "Just 1 please" ;-)

Cheers
Stu
